### ISSUE TITLE
KeyError in templates/reporting.html line 249

### ISSUE THREAD

**Issue by jdubp** [Kart1keya/Hachi#1]
Hi,

Great work on the tool! When running a sample however, I receive an internal server error on the reporting lookup... traceback <type 'exceptions.KeyError'> at /report/* line 249

$for entry in table_data[0]

The error is due to the the table_data dict not being populated. In reviewing the code in hachi.py, table_data becomes populated only if certain JSON files exist in the OUTPUT directory. In my case, the OUTPUT directory for the sample gets created but does not contain the JSON files. It appears the JSON files are supposed to created in the OUTPUT directory from execution of the scanda.py file? 

Any suggestions?

**Comment by Kart1keya** [comment]
@jdubp Yes, this exception is due to nonpresence of JSON files in the output directory. For the creation of JSON files, you'll have to run scanda.py as well as mentioned in Step 5 in the readme file:
`Run hachi.py and scanda.py`

scanda.py is the core engine for scanning the sample. Hope this helps. :)

**Comment by jdubp** [comment]
Thank you for the response. I should have mentioned that I receive this error with hachi.py and scanda.py both running. I was attempting to do my own troubleshooting. 

The current environment I am using for testing is Windows 7 and Python 2.x. I have validated the MSMQ private pipe is functioning. When i submit a sample it lives in the "Pending" state in the GUI and the error is generated upon attempting to click the sample to get the report. It may be an environment issue but it appears samples do not actually run in my environment.
